rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2254,7 +2254,14 @@ def test_upgrade_db_41_to_42(user_data_dir, messages_aggregator):
         assert table_exists(cursor, 'calendar_reminders') is False
         assert cursor.execute('SELECT MAX(seq) FROM location').fetchone()[0] == 45
         assert cursor.execute('SELECT COUNT(tx_hash) FROM balancer_events').fetchone()[0] == 2
-        assert cursor.execute('SELECT COUNT(*) from used_query_ranges').fetchone()[0] == 2
+        assert cursor.execute('SELECT name from used_query_ranges').fetchall() == [
```

Comment:
Can you also add some unrelated used query ranges in the DB? And test that they are not deleted. This is always a tricky test to get right.